software/acpid.txt

```diff
@@ -0,0 +1,123 @@
+ACPID
+________________________________________________________________________________
+
+
+Prerequisites
+________________________________________________________________________________
+
+Make sure that ACPI-related kernel options are enabled. [0]
```

Comment:
It'd be great to include the kernel configuration information here. See https://k1ss.org/wiki/kernel/config for formatting.